src/main.js

```diff
@@ -122,20 +126,19 @@ function createNewIdea() {
   var currentIdea = new Idea(userNewTitle.value, userNewBody.value);
   if (userNewTitle.value && userNewBody.value) {
   savedIdeas.push(currentIdea);
+  saveIdeaToStorage();
   saveIdeaButton.disabled = true;
-  // saveIdeaToStorage();
   }
 }
 
-// function saveIdeaToStorage() {
-//   localStorage.setItem('ideas', JSON.stringify(savedIdeas));
-// }
-//
-// function retrieveMadeIdeaCards() {
-//   savedIdeas = JSON.parse(localStorage.getItem('ideas')) || [];
-//
-//   showUsersIdeaCard();
-// }
+function saveIdeaToStorage() {
```

Comment:
These are the functions saving to storage and retrieving from storage. The retrieve function only fire on page reload. The storing function happens each time we make a change to our data model (the savedIdeas array).